compiler/rustc_infer/src/infer/error_reporting/mod.rs

```diff
@@ -1486,25 +1486,21 @@ impl<'a, 'tcx> InferCtxt<'a, 'tcx> {
                     let kind = key.descr();
                     let mut returned_async_output_error = false;
                     for sp in values {
```

Comment:
I know it's not your code, but while you're modifying this code, could you change the name of `values` to `spans`, and `key` to `ty_category`? At the moment, the variable names are undescriptive.

Reply:
Done!